nsv13/code/modules/overmap/radar.dm

```diff
@@ -137,6 +137,8 @@
 					thecolour = "#00FFFF"
 				else if(OM.faction == linked.faction)
 					thecolour = "#32CD32"
+				else if(OM.faction == "planet")
```

Comment:
This means AI ships will warcrime planets to death rather than focusing on the real targets. Ensure planets are added to the warcrime_blacklist!